addons/l10n_ro_efactura/models/account_move.py

```diff
@@ -0,0 +1,130 @@
+import requests
+
+from odoo import models, fields, _, api
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_ro_edi_document_ids = fields.One2many(
+        comodel_name='l10n_ro_edi.document',
+        inverse_name='invoice_id',
+    )
+    l10n_ro_edi_active_document_id = fields.Many2one(
+        comodel_name='l10n_ro_edi.document',
+        compute='_compute_from_l10n_ro_edi_document_ids',
+    )
+    l10n_ro_edi_state = fields.Selection(
+        selection=[
+            ('invoice_sending', 'Sending'),
+            ('invoice_sending_failed', 'Error'),
```

Comment:
I think we should only store valid state on the invoice: sending & sent here.
Suppose the web service is down. We only want to retry the web service... not to mark all invoices as in error. The document is there to give you more details.